invenio_app_rdm/theme/templates/semantic-ui/invenio_app_rdm/header_frontpage.html

```diff
@@ -12,6 +12,7 @@
 {#- Remove search field from top bar on frontpage #}
 {%- block navbar_search %}{% endblock %}
 
+
```

Comment:
Maybe revert this change to keep the commit shorter?

Reply:
i agree!